pymemcache/test/utils.py

```diff
@@ -43,6 +43,10 @@ def __init__(self,
         self.ignore_exc = ignore_exc
         self.encoding = encoding
 
+    def clear(self):
+        '''Method used to clear/reset mock cache'''
```

Comment:
We use `"` for docblocks elsewhere in the code. Let's stay consistent.

```suggestion
        """Method used to clear/reset mock cache"""
```